python-frontend/src/test/java/org/sonar/python/semantic/v2/SymbolTableBuilderV2Test.java

```diff
@@ -49,89 +106,69 @@ def script_do_something(param):
     }
 
     {
-      var assignmentStatement = (AssignmentStatement) fileInput.statements().statements().get(3);
+      var assignmentStatement = (AssignmentStatement) statements.get(3);
       var bName = (Name) assignmentStatement.children().get(0).children().get(0);
-      var bNameSymbol = bName.symbolV2();
-      Assertions.assertNotNull(bNameSymbol);
-      Assertions.assertEquals("b", bNameSymbol.name());
-      Assertions.assertEquals(1, bNameSymbol.usages().size());
+      assertNameSymbol(bName, "b", 1);
 
       var aName = (Name) assignmentStatement.children().get(2).children().get(0).children().get(0);
-      var aNameSymbol = aName.symbolV2();
-      Assertions.assertNotNull(aNameSymbol);
-      Assertions.assertEquals("a", aNameSymbol.name());
-      Assertions.assertEquals(2, aNameSymbol.usages().size());
+      assertNameSymbol(aName, "a", 2);
     }
 
     {
-      var assignmentStatement = (AssignmentStatement) fileInput.statements().statements().get(4);
+      var assignmentStatement = (AssignmentStatement) statements.get(4);
       var xName = (Name) assignmentStatement.children().get(0).children().get(0);
-      var xNameSymbol = xName.symbolV2();
-      Assertions.assertNotNull(xNameSymbol);
-      Assertions.assertEquals("x", xNameSymbol.name());
-      Assertions.assertEquals(1, xNameSymbol.usages().size());
+      assertNameSymbol(xName, "x", 1);
     }
 
     {
-      var functionDef = (FunctionDef) fileInput.statements().statements().get(5);
+      var functionDef = (FunctionDef) statements.get(5);
       var scriptFunctionName = (Name) functionDef.name();
-      var scriptFunctionNameSymbol = scriptFunctionName.symbolV2();
-      Assertions.assertNotNull(scriptFunctionNameSymbol);
-      Assertions.assertEquals("script_do_something", scriptFunctionNameSymbol.name());
-      Assertions.assertEquals(2, scriptFunctionNameSymbol.usages().size());
+      assertNameSymbol(scriptFunctionName, "script_do_something", 2);
     }
 
     {
-      var functionCallName = (Name) fileInput.statements().statements().get(6).children().get(0).children().get(0);
-      var functionCallNameSymbol = functionCallName.symbolV2();
-      Assertions.assertNotNull(functionCallNameSymbol);
-      Assertions.assertEquals("script_do_something", functionCallNameSymbol.name());
-      Assertions.assertEquals(2, functionCallNameSymbol.usages().size());
+      var functionCallName = (Name) statements.get(6).children().get(0).children().get(0);
+      assertNameSymbol(functionCallName, "script_do_something", 2);
     }
 
   }
+
+  private static void assertNameSymbol(Name functionCallName, String script_do_something, int expected) {
+    var functionCallNameSymbol = functionCallName.symbolV2();
+    Assertions.assertThat(functionCallNameSymbol).isNotNull();
+    Assertions.assertThat(functionCallNameSymbol.name()).isEqualTo(script_do_something);
+    Assertions.assertThat(functionCallNameSymbol.usages()).hasSize(expected);
```

Comment:
I guess you forgot to update variable names here?
If I understand correctly, the first parameter can be any name (as in tree)?
Second parameter is the expected symbol name and third parameter is expectedUsageCount?